pkg/utils/managedresources/managedresources.go

```diff
@@ -263,14 +270,14 @@ func WaitUntilListDeleted(ctx context.Context, client client.Client, mrList *res
 }
 
 // WaitUntilDeleted waits until the given managed resource is deleted.
-func WaitUntilDeleted(ctx context.Context, client client.Client, namespace, name string) error {
+func WaitUntilDeleted(ctx context.Context, clientPar client.Client, namespace, name string) error {
```

Comment:
```suggestion
func WaitUntilDeleted(ctx context.Context, c client.Client, namespace, name string) error {
```

I think usually `c` is used as var name accross the when client is already taken.